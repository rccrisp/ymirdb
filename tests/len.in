set a 1
set b a
set c b
set d b c
set e d
len a
len b
len c
len d
len e
bye
