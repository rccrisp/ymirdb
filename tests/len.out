> ok

> ok

> ok

> ok

> ok

> 1

> 1

> 1

> 2

> 2

> bye
